src/sentry/tasks/deletion.py

```diff
@@ -389,3 +389,42 @@ def delete_repository(object_id, transaction_id=None, actor_id=None, **kwargs):
             },
             countdown=15,
         )
+
+
+@instrumented_task(
+    name='sentry.tasks.deletion.delete_organization_integration',
+    queue='cleanup',
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES
+)
+@retry(exclude=(DeleteAborted, ))
+def delete_organization_integration(object_id, transaction_id=None, actor_id=None, **kwargs):
+    from sentry import deletions
+    from sentry.models import OrganizationIntegration
+
+    try:
+        instance = OrganizationIntegration.objects.get(id=object_id)
+    except OrganizationIntegration.DoesNotExist:
+        return
+
+    if instance.status == ObjectStatus.VISIBLE:
+        raise DeleteAborted
+
+    task = deletions.get(
+        model=OrganizationIntegration,
+        actor_id=actor_id,
+        query={
+            'id': object_id,
+        },
+        transaction_id=transaction_id or uuid4().hex,
+    )
```

Comment:
Does `deletions.get` actually do the deleting? Just curious.

Reply:
No, the next line, `task.chunk()` does the work.